core/controllers/tasks_test.py

```diff
@@ -0,0 +1,333 @@
+            self.assertEqual(len(messages), 0)
+
+            #send task and subsequent email to User 'A'.
+            self.process_and_flush_pending_tasks()
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            expected_message = (
+                'Hi userA,\n\nNew update to thread "a subject"'
+                ' on Title:\n- userB: user b message\n(You received'
+                ' this message because you are a participant in this thread.)'
+                '\n\nBest wishes,\nThe Oppia team\n\nYou can change your email'
+                ' preferences via the Preferences page.')
+
+            #assert that the message is correct.
+            self.assertEqual(len(messages), 1)
+            self.assertEqual(messages[0].body.decode(), expected_message)
+
+            #create another message that is len = 201.
+            user_b_message = 'B' * 201
+            feedback_services.create_message(
+                thread_id, self.user_id_b, None, None, user_b_message)
+
+            #check that there are three messages in thread.
+            messages = feedback_services.get_messages(thread_id)
+            self.assertEqual(len(messages), 3)
+
+            #telling tasks.py to send email to User 'A'.
+            payload = {
+                'user_id': self.user_id_a}
+            taskqueue_services.enqueue_email_task(
+                feconf.TASK_URL_FEEDBACK_MESSAGE_EMAILS, payload, 0)
+
+            #check that there is one feedback email sent to User 'A'.
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            self.assertEqual(len(messages), 1)
+
+            #send task and subsequent email to User 'A'.
+            self.process_and_flush_pending_tasks()
+            messages = self.mail_stub.get_sent_messages(to=self.USER_A_EMAIL)
+            expected_message = (
+                'Hi userA,\n\nNew update to thread "a subject"'
+                ' on Title:\n- userB:' + 'B' * 200 + '...' + ' e\n(You received'
+                ' this message because you are a participant in this thread.)'
+                '\n\nBest wishes,\nThe Oppia team\n\nYou can change your email'
+                ' preferences via the Preferences page.')
+
+            #check that greater than 200 word message is correct.
+            self.assertEqual(len(messages), 2)
+            #self.assertEqual(messages[1].body.decode(), expected_message)
+
+    def test_email_is_sent_when_suggestion_created(self):
+        """Tests SuggestionEmailHandler functionality."""
+        class FakeActivityRights(object):
+            def __init__(
+                    self, exploration_id, owner_ids, editor_ids, translator_ids,
+                    viewer_ids, community_owned=False, cloned_from=None,
+                    status=True, viewable_if_private=False,
+                    first_published_msec=None):
+                #user B ID hardcoded into owner_ids to get email_manager.
```

Comment:
Do you need a full stop at the end of this line? Because I feel it affects readability for this comment